account/client.go

```diff
@@ -7,7 +10,7 @@ import (
 	"github.com/stripe/stripe-go/form"
 )
 
-// Client is used to invoke /account APIs.
+// Client is used to invoke APIs related to accounts.
```

Comment:
is the change before /account is not a valid comment?

Reply:
Yeah, I made this mostly because I feel a little like the particular URLs being accessed by the package are an implementation detail — not necessarily something that needs to be included in the documentation. The docs are still overly terse, but I was intending to basically replace a URL with a resource name.